mount.go

```diff
@@ -299,10 +299,20 @@ func virtioBlkStorageHandler(_ context.Context, storage pb.Storage, s *sandbox)
 	// If hot-plugged, get the device node path based on the PCI address else
 	// use the virt path provided in Storage Source
 	if strings.HasPrefix(storage.Source, "/dev") {
-
+		// waiting for a pmem/nvdimm device
```

Comment:
Why did you go with adding this logic in the virtioBlkStorageHandler @devimc ?
How about having a separate handler for pmem devices?

Reply:
because it's the same logic that we have in the runtime

https://github.com/kata-containers/runtime/blob/master/virtcontainers/qemu.go#L1031-L1048

`hotplugAddBlockDevice` -> `ExecuteNVDIMMDeviceAdd`

Reply:
@amshinde  ignore my comment, I know what you mean